content/docs/glossary/minimum-balance.mdx

```diff
@@ -24,6 +26,10 @@ The absolute minimum balance for an account is 1 XLM, which is equal to `(2 + 0
 
 For example, an account with 1 trustline and 2 offers would have a minimum balance of `(2 + 3 entries) * 0.5 base reserve = 2.5 XLM`.
 
+[Sponsored Reserves](./sponsored-reserves.mdx) will affect the # of sponsoring entries and # of sponsored entries.
+
+[Claimable Balances](./claimable-balance.mdx) are reflected in the # of sponsoring entries, so each claimable balance will require an additional 0.5 XLM. For example, an account that creates a claimable balance would have a minimum balance of `(2 + 0 entries + 1 sponsoring entry) * 0.5 base reserve = 1.5 XLM`.
```

Comment:
Might want to say "claimable balance with 1 claimant", because this relationship isn't true in general.

Reply:
Good catch. Added claimants to the explanation.